common/djangoapps/enrollment/views.py

```diff
@@ -278,6 +281,44 @@ def get(self, request, course_id=None):
             )
 
 
+class EnrollmentCourseRosterView(APIView, ApiKeyPermissionMixIn):
+
+    authentication_classes = OAuth2AuthenticationAllowInactiveUser, EnrollmentCrossDomainSessionAuth
+    permission_classes = ApiKeyHeaderPermissionIsAuthenticated,
+
+    @method_decorator(ensure_csrf_cookie_cross_domain)
+    def get(self, request, course_id=None):
+
+        if not course_id:
```

Comment:
Do we need this `if` block?
What happens in the call below to `CourseKey.from_string` if it's passed a `None`/`falsey` value?

Assuming the `try` block handles it well, we could remove this block entirely.